purchase_request_department/models/purchase_request.py

```diff
@@ -0,0 +1,38 @@
+# Copyright 2017-2020 Forgeflow S.L.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo import api, fields, models
+
+
+class PurchaseRequest(models.Model):
+    _inherit = "purchase.request"
+
+    def _get_my_department(self):
+        employees = self.env.user.employee_ids
+        return (
+            employees[0].department_id
```

Comment:
Same  for`department_id`  https://github.com/odoo/odoo/blob/b49a23fdd83c139f19c22017667005de2dd8cc75/addons/hr/models/res_users.py#L92